packages/catalog-search/package.json

```diff
@@ -58,7 +58,7 @@
     "react-test-renderer": "17.0.2"
   },
   "peerDependencies": {
-    "@edx/frontend-platform": "^7.0.0",
+    "@edx/frontend-platform": "^7.0.0 || ^8.0.0",
```

Comment:
[nit/sanity check] Will there be consuming MFEs still installing on frontend-platform v7 once these packages are upgraded in the consuming MFEs? If not, I'm wondering if frontend-platform v7 could be dropped in favor of only supporting v8 (though, doing so would require these packages to be released as a breaking change otherwise)?

Reply:
Thank you touching on this, your insights will be invaluable here. All MFEs will be moved to v8 once these packages are upgraded in the consuming MFEs. The initial proposal was indeed to transition all MFEs directly to `frontend-platform `v8, which would have necessitated a major version release due to breaking changes, which was opposed & there was a strong recommendation to maintain compatibility with frontend-platform v7 in order to avoid breaking change releases for the few MFEs having consumers. ([[1]](https://github.com/openedx/frontend-component-header/pull/369#discussion_r1574802668), [[2]](https://github.com/openedx/frontend-component-footer/pull/317#discussion_r1574799566), [[3]](https://github.com/edx/frontend-component-footer-edx/pull/366#discussion_r1575740104)).

@abdullahwaheed has been a strong advocate of avoiding major version releases in this context & can elaborate on how the decision to prevent major version releases in this case, by maintaining frontend-platform v7 as a peer dependency, outweighs the benefits that might come from moving exclusively to v8 for a case where all MFEs are to be moved to v8 eventually.

As we continue with the implementation, as detailed in all related PRs, all MFEs will indeed be moved to `frontend-platform` v8 & `frontend-build` v14 as part of this coordinated upgrade process.